### PR TITLE
feature--repo-list-layout

### PR DESCRIPTION
**Pull request by jeffworley**

### CODE REVIEW

**Comment by ericksonk** [comment]
Good start :+1:

**Comment by jeremyrist** [comment]
:+1: attaboy!

**Comment by s4dmw** [comment]
lookin good. i forsee so much css in our future...  :+1:

File: src/index.html [hunk 1]
```diff
@@ -58,38 +58,82 @@ <h3 id="username">{{login}}</h3>
...
+                  <li><a>Sources</a></li>
+                  <li><a>Forks</a></li>
+                  <li><a>Mirrors</a></li>
                 </ul>
               </nav>
             </section>
+            <section id="repos">
+              <ul id="repo-list">
+                <li>
+                  <div>
```
Comment (kellymurray) [inline src/index.html:78]: so many `divs`... :smile:

**Pull request merged**